torch/csrc/distributed/rpc/utils.cpp

```diff
@@ -220,6 +220,9 @@ c10::List<at::Tensor> cloneSparseTensors(
   // force a clone(). Some Tensors are effectively small views, only using
   // ~1% of the underlying Storage.
   auto worthRecopying = [](const at::Tensor& t) -> bool {
+    if (!t.has_storage()) {
+      return false; // avoid throwing below.
```

Comment:
BTW, as JIT does not support sparse tensors yet, what error will user see after this change?

Reply:
When I ran this last night, they see the error from JIT Pickler, rather than this line, where we were looking at the storage without it being there.